exporter/dynatraceexporter/metrics_exporter.go

```diff
@@ -136,13 +136,18 @@ func (e *exporter) serializeMetrics(md pdata.Metrics) []string {
 				metricLines, err := serialization.SerializeMetric(e.settings.Logger, e.cfg.Prefix, metric, e.defaultDimensions, e.staticDimensions, e.prevPts)
 
 				if err != nil {
-					e.settings.Logger.Sugar().Errorf("failed to serialize %s %s: %s", metric.DataType().String(), metric.Name(), err.Error())
+					e.settings.Logger.Sugar().Errorw(
+						"failed to serialize",
+						"datatype", metric.DataType().String(),
+						"name", metric.Name(),
+						zap.Error(err),
+					)
 				}
 
 				if len(metricLines) > 0 {
 					lines = append(lines, metricLines...)
 				}
-				e.settings.Logger.Debug(fmt.Sprintf("Serialized %s %s - %d lines", metric.DataType().String(), metric.Name(), len(metricLines)))
+				e.settings.Logger.Sugar().Debugf("Serialized %s %s - %d lines", metric.DataType().String(), metric.Name(), len(metricLines))
```

Comment:
This is completely personal preference but I am rather against formatted strings being used as the log message, simply because it means searching logs require ether prefix matching which may also match against another message that is unrelated or the use of wildcards which means the search tool has to spend more time trying to match the regex. 

This completely optional but I would emit the log like so:
```suggestion
e.settings.Logger.Sugar().Debug("Serialized metric data", 
    "metric-type": metric.DataType().String(),
    "metric-name": metric.Name(),
    "data-len": len(metricLines),
) 
```